knowledge-base/creating-tablecells-with-gotoaction-and-uriaction.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Creating TableCells with GoToAction and UriAction
+description: This article demonstrates how can create table cells with GoTo action and UriAction in PDF document with Telerik Document Processing.
+type: how-to 
+page_title: Creating TableCells with GoToAction and UriAction
+slug: 
+position: 0
+tags: pdf, datatable, cells, widget
+res_type: kb
+---
+
+|Product Version|Product|Author|
+|----|----|----|
+|N/A|Telerik Document Processing|Maria Terzieva|
+
+## Description
+
+The pdf format is different than the format in MS Word or other commonly used editors. With the PDF format, each content element has a specific position, and in some cases, each character is positioned separately. PDF standard does not have any information about tables as well, once exported the tables in the document are represented by lines and text fragments. 
```

Comment:
The first sentence does bring any value, can be removed. extend te description a bit more and describe the proposed solution.